run_testing_worker.py

```diff
@@ -14,6 +14,8 @@
 from google.cloud.spanner_v1 import Client
 
 REGION = "regional-us-central1"
+WORKER_INDEX = int(os.getenv("DJANGO_WORKER_INDEX", 0))
```

Comment:
I think you could use a `UUID` here to (practically) guarantee no clashes will happen.